projects/smoldot-extension/src/components/Tab.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import {
+    Grid,
+    Typography,
+    Box,
+    Accordion,
+    AccordionSummary,
+    AccordionDetails } from '@material-ui/core';
+import { Switch } from '.';
+import { TabInterface, uApp, Networks } from '../utils/types';
+import { IconWeb3 } from '../components';
```

Comment:
`{ Switch }` and `{ IconWeb3 }` are exported from the same place. Should it be done in one preferable go?

Reply:
Absolutely - that was probably an auto-correct with copy/pasting stuff